### DISCUSSION TITLE
installation DLL injector problem

### DISCUSSION THREAD

**Discussion by Dashtails750** [winstxnhdw/lc-hax#37] [Q&A] [answered]
when I use the requirements.bat I get this, then when i use the launch.bat it says cannot find injector DLL
![Screenshot (108)](https://github.com/winstxnhdw/lc-hax/assets/154655516/2021fbbd-70dd-4ee0-9af4-9f25d15f733a)

**Comment by winstxnhdw** [comment, marked as answer]
Is your antivirus removing the injector?

**Reply by Dashtails750** [reply]
i think so
![Screenshot (109)](https://github.com/winstxnhdw/lc-hax/assets/154655516/48224561-da09-4af7-a8da-e8f9617fa97e)
how do i fix this??

**Reply by Dashtails750** [reply]
i got is working again by  pressing restore and running the requirement.bat again thanks
